yjit/src/backend/x86_64/mod.rs

```diff
@@ -26,6 +26,15 @@ pub const _C_ARG_OPNDS: [Opnd; 6] = [
     Opnd::Reg(R9_REG)
 ];
 
+pub const _C_ARG_OPNDREGS: [X86Opnd; 6] = [
+    X86Opnd::Reg(RDI_REG),
+    X86Opnd::Reg(RSI_REG),
+    X86Opnd::Reg(RDX_REG),
+    X86Opnd::Reg(RCX_REG),
+    X86Opnd::Reg(R8_REG),
+    X86Opnd::Reg(R9_REG)
+];
```

Comment:
Maybe we could avoid defining these twice, use C_ARG_REGS[idx].unwrap_reg() when the reg is needed.

Reply:
Given that we only seem to use them *outside* this file in gen_entry_prologue (and only two of them), it's possible we don't need them X86Opnd::Reg form of them at all.

For now I'll do the unwrap/rewrap, but the longer-term answer may be "export less" and then we can just keep it in this file's preferred representation.